web_m2x_options/static/src/components/form.esm.js

```diff
@@ -0,0 +1,402 @@
+/** @odoo-module **/
+import {registry} from "@web/core/registry";
+import {
+    Many2ManyTagsField,
+    Many2ManyTagsFieldColorEditable,
+    many2ManyTagsField,
+} from "@web/views/fields/many2many_tags/many2many_tags_field";
+import {Many2OneField, many2OneField} from "@web/views/fields/many2one/many2one_field";
+import {FormController} from "@web/views/form/form_controller";
+import {
+    KanbanMany2OneAvatarField,
+    Many2OneAvatarField,
+} from "@web/views/fields/many2one_avatar/many2one_avatar_field";
+import {Many2ManyTagsAvatarField} from "@web/views/fields/many2many_tags_avatar/many2many_tags_avatar_field";
+
+import {Many2XAutocomplete} from "@web/views/fields/relational_utils";
+import {evaluateBooleanExpr} from "@web/core/py_js/py";
+import {isX2Many} from "@web/views/utils";
+import {patch} from "@web/core/utils/patch";
+import {session} from "@web/session";
+
+function is_option_set(option) {
+    if (typeof option === "string") return option === "true" || option === "True";
+    if (typeof option === "boolean") return option;
+    return false;
+}
+
+Many2OneField.props = {
+    ...Many2OneField.props,
+    noSearchMore: {type: Boolean, optional: true},
+    fieldColor: {type: String, optional: true},
+    fieldColorOptions: {type: Object, optional: true},
+};
+Many2XAutocomplete.props = {
+    ...Many2XAutocomplete.props,
+    fieldColor: {type: String, optional: true},
+    fieldColorOptions: {type: Object, optional: true},
+};
+
+KanbanMany2OneAvatarField.props = {
+    ...KanbanMany2OneAvatarField.props,
+    fieldColor: {type: String, optional: true},
+    fieldColorOptions: {type: Object, optional: true},
+};
+
```

Comment:
```suggestion

KanbanMany2OneAvatarUserField.props = {
    ...KanbanMany2OneAvatarUserField.props,
    noSearchMore: {type: Boolean, optional: true},
    fieldColor: {type: String, optional: true},
    fieldColorOptions: {type: Object, optional: true},
};

Many2OneAvatarUserField.props = {
    ...Many2OneAvatarUserField.props,
    noSearchMore: {type: Boolean, optional: true},
    fieldColor: {type: String, optional: true},
    fieldColorOptions: {type: Object, optional: true},
};

```

This is a fix for yet another O.W.L. props validation error (on most recent Odoo source code). This time, the error is raised on opening the PoS module in developer mode.

This module is in need of a bigger rework to tackle all these separate cases.

Edit: added more necessary missing props, this time for `Many2OneAvatarUserField`.